.github/workflows/nightly-branch-maker.yaml

```diff
@@ -0,0 +1,24 @@
+name: nightly-test branch creation
+# To help with triage, if we're waiting for a response from a
+# user we label the issue waiting-response.
+# When a user comments on the issue the label is removed.
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create-branch:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
```

Comment:
I thought this was changed to use the replacement for set-output? Has this PR lost some changes we've made in previous review cycles?

Reply:
Oh yeah this file needs to be deleted right? Because .github/workflows/teamcity-nightly-workflow.yaml contains this now?

Reply:
yeah i didn't realize it wasnt removed when the file name was updated. I'll fix that right now.